assets/css/omgf-frontend.css

```
@@ -89,6 +89,23 @@
 	}
 }
 
+#wp-admin-bar-omgf.dot.info > .ab-item:after {
+	background-color: #0074D9;
+	-webkit-animation: blinking-info 2.5s; /* Safari 4+ */
+	-moz-animation: blinking-info 2.5s; /* Fx 5+ */
+	-o-animation: blinking-info 2.5s; /* Opera 12+ */
+	animation: blinking-info 2.5s; /* IE 10+, Fx 29+ */
+}
+
+@-webkit-keyframes blinking-info {
```

Stylelint (17.3.0)

[error] Unexpected vendor-prefixed at-rule "@-webkit-keyframes" (at-rule-no-vendor-prefix)

(at-rule-no-vendor-prefix)